### PR TITLE
Add backplane audit log

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: deploy/crds/splunkforwarder_v1alpha1_splunkforwarder_cr.yaml [hunk 1]
```diff
@@ -16,3 +16,7 @@ spec:
     index: openshift_managed_debug_node
     whitelist: \.log$
     sourcetype: openshift:debug
+  - path: /host/var/lib/kubelet/pods/*/volumes/kubernetes.io~*/pvc-*/backplane-audit.log
```
Comment (jaybeeunix) [inline deploy/crds/splunkforwarder_v1alpha1_splunkforwarder_cr.yaml:19]: You don't need to make this change, as this sample file isn't actually used.

**Comment by jaybeeunix** [comment]
Other than the sample file, everything looks like what we talked about (for others: making a copy of the "normal" splunk config for the hives).

**Comment by wanghaoran1988** [comment]
@jaybeeunix  Comments addressed.

**Comment by jaybeeunix** [comment]
thanks!

/lgtm

**Pull request merged**